types/src/sector/mod.rs

```diff
@@ -1,13 +0,0 @@
-// Copyright 2019-2022 ChainSafe Systems
-// SPDX-License-Identifier: Apache-2.0, MIT
-
-pub mod post;
```

Comment:
How abt this post.rs?

Reply:
It was renamed to `utils/json/src/sector.rs`.